.github/workflows/test-everything.yml

```diff
@@ -182,3 +186,19 @@ jobs:
         run: |
           arbor-build-catalogue -v default mechanisms/default
           ./scripts/test-catalogue.py ./default-catalogue.so
+  testpip:
+    name: "Pip build test"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Python packages
+        run:  pip install numpy setuptools scikit-build ninja cmake
+      - name: Clone w/ submodules
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Build and install Arbor using pip
+        run: python3 -m pip install --verbose --install-option="-DARB_VECTORIZE=ON" --install-option="-DARB_ARCH=native" .
```

Comment:
Do we verify that vectorization is actually enabled?

Reply:
We do now :)

Reply:
Cool :)